drivers/bluetooth/hci/h5.c

```diff
@@ -566,7 +576,7 @@ static void bt_uart_isr(const struct device *unused, void *user_data)
 			 */
 			if (H5_HDR_RELIABLE(hdr) &&
 			    H5_HDR_SEQ(hdr) != h5.tx_ack) {
-				BT_ERR("Seq expected %u got %u. Drop packet",
+				LOG_ERR("Seq expected %u got %u. Drop packet",
 				       h5.tx_ack, H5_HDR_SEQ(hdr));
```

Comment:
Maybe add space here better for alignment

Reply:
This will be a general comment if it is not done everywhere:

For all multiline `BT_<XXX>" statements, the next line should have been indented by 7 spaces. With `LOG_<XXX>", this indent is now 8 spaces, which shall be converted to a tab.

Reply:
Good point. I thought git-clang-format would fix this. But I fell into a trap: The second line there was not modified, so it was not touched by git-clang-format. Fixed now.